tms34020: implement CLIP opcode

plaes opened the pull request:
This pull request should is supposed to implement CLIP opcode for tms34020.

happppp commented:
If you want to do such verbose logging, MAME has logmacro. I don't think it's necessary to log every CLIP in detail though, so I'd just remove them all. Also, you forgot to clear V.

plaes commented:
> If you want to do such verbose logging, MAME has logmacro. I don't think it's necessary to log every CLIP in detail though, so I'd just remove them all. Also, you forgot to clear V.

Thanks, dropped the extra logging and fixed the `V` flag handling.
